pkg/solidity-utils/contracts/math/WeightedMath.sol

```diff
@@ -66,7 +66,7 @@ library WeightedMath {
     // The invariant is used to collect protocol swap fees by comparing its value between two times.
     // So we can round always to the same direction. It is also used to initiate the BPT amount and,
     // because there is a minimum BPT, we round down the invariant.
```

Comment:
Stale comment now. (And not well-written besides.) The first sentence is left-over from v2.
```suggestion
    // The invariant functions are called by the Vault during various liquidity operations, and require a specific
    // rounding direction in order to ensure safety (i.e., that the final result is always rounded in favor of the protocol).
```